### ISSUE TITLE
There are too many angular 2 seed projects - What's this project for?

### ISSUE THREAD

**Issue by silentHoo** [angular/angular2-seed#82]
I'm working for a while on a large project which we started some weeks ago with the release of angular ~beta.16.

Before we started we wrote our concepts for the application architecture and started searching for a project which gives us a handful of useful tools to help us build something maintainable on a best practice approach. The problem we now facing is a bit difficult, we have now at least three options we can chose from:
- https://github.com/angular/angular2-seed
- https://github.com/mgechev/angular2-seed
- https://github.com/angular/angular-cli

Could you please provide your plans for the future how we can classify this official _angular2-seed_ project? Will it be integrated into the _angular-cli_? It's difficult for angular2-starters to find the right track. Thank you for your great work!

**Comment by bbottema** [comment]
This is the only angular seed that uses webpack and it has been a great help to me to get up and running using Angular2, TypeScript and Webpack.

The other popular Angular2 Webpack seed worth mentioning, [/AngularClass/angular2-webpack-starter](https://github.com/AngularClass/angular2-webpack-starter), proved to be too hard to understand, because there is so much 'magic' going on in the config files.

**Comment by silentHoo** [comment]
Okay, that sounds comprehensible to me. Thank you @bbottema for your answer here. Interesting that there is another starter pack for that :dizzy_face:

But what are the `best practice` plans of the angular core team for the future? Could someone of the core team please provide some additional information or statements about that?

**Comment by nfantone** [comment]
I think this a completely valid question. It'd be nice to have some "official" input on this.

**Comment by pokakhoaitay** [comment]
If you refer to use webpack in your projects, i think you can check this out: [https://angular.io/docs/ts/latest/guide/webpack.html](url).
It very easy to understand and clearly. You can build your own a starter project and fun with it.

**Comment by railsr** [comment]
Can someone explain how to add bootstrap and jquery to this project?

I just install it through npm and connect from `vendor.ts` with import `bootstrap/dist/css/bootstrap.min.css;` But I'm not sure if this is correct.

And I also get `Uncaught Error: Bootstrap's JavaScript requires jQuery` after adding `bootstrap/dist/js/bootstrap.min.js;` to vendor. 

I tried to connect jquery the same way but it didn't work.

**Comment by gforceg** [comment]
That link points to https://github.com/angular/angular2-seed/issues/url
...
